install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Installing repositories...${LOG_END}"
+
+./install-repositories.sh
+
+cd keep-subgraph
+
+printf "${LOG_START}Running subgraph scripts...${LOG_END}"
+
+CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY} ETH_RPC_URL=${ETH_RPC_URL} SUBGRAPH_DEPLOY_KEY=${SUBGRAPH_DEPLOY_KEY} SUBGRAPH_SLUG=${SUBGRAPH_SLUG} ./deploy-subgraph.sh
```

Comment:
Does it mean that running `install.sh` script locally will update the version that is deployed to thegraph.com?
I would expect that `install.sh` sets up allthekeeps to be run locally, it shouldn't publish anything from dev machine to thegraph.

The script is not used in the github workflow, what is the use case for it? Maybe we could remove it not to cause confusion. If we need to set up local instance we could handle it in a separate PR, as this would require setting local graph node or using already published subgraph.